configs/twilio_paste.json

```
@@ -16,8 +16,14 @@
     "lvl5": "main h6",
     "text": "main p, main li"
   },
+  "selectors_exclude": [
+    "#page-aside *"
```

```suggestion
    "#page-aside"
```
This remove the whole element.